src/inverseRobot.js

```diff
@@ -7,7 +7,18 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const clone = {};
+
+  for (const key in robot) {
+    const newKey = robot[key];
+
+    if (clone.hasOwnProperty(newKey)) {
```

Comment:
Consider using 'Object.prototype.hasOwnProperty.call(clone, newKey)' instead of 'clone.hasOwnProperty(newKey)' to avoid issues that may arise if the object has a property with the name 'hasOwnProperty'. This is a more robust way to check for property existence.